### PR TITLE
Clean up «Installer Kubernetes avec Minikube»

### PR DESCRIPTION
**Pull request by sftim**
@remyleone I spotted some nits in the [French localization ](https://kubernetes.io/fr/docs/setup/learning-environment/minikube/)of https://kubernetes.io/docs/setup/learning-environment/minikube/; I hope these changes help tidy them up. ([preview](https://deploy-preview-17945--kubernetes-io-master-staging.netlify.com/fr/docs/setup/learning-environment/minikube/))

Fixup for #17939 
/kind cleanup
/priority backlog

### CODE REVIEW

File: content/fr/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -226,11 +226,11 @@ minikube start --kubernetes-version {{< param "fullversion" >}}
...
 #### Spécification du pilote de machine virtuelle
 
-Vous pouvez changer le pilote de machine virtuelle en ajoutant l'indicateur `--vm-driver = <nom_du_processeur>` à `minikube start`.
-Par exemple, la commande serait.
+Vous pouvez changer le pilote de machine virtuelle en ajoutant l'indicateur `--vm-driver=<nom_du_processeur>` à `minikube start`.
+Par exemple, la commande serait:
 
 ```shell
-minikube start --vm-driver=<driver_name>
+minikube start --vm-driver=<nom_de_processeur>
```
Comment (remyleone) [inline content/fr/docs/setup/learning-environment/minikube.md]: Nom du pilote

**Comment by remyleone** [comment]
@sftim Thanks for fixing that up :) I cannot see the page on the website in French though.

**Comment by sftim** [comment]
It does look like the deploy might be broken. Separate issue?

**Comment by remyleone** [comment]
Sure but I don't have admin rights on the netlify.

Le jeu. 5 déc. 2019 à 12:25, Tim Bannister <notifications@github.com> a
écrit :

**Comment by sftim** [comment]
These fixes should be OK to approve @remyleone; the issue with deployment was not related.

**Pull request merged**